gui/wxpython/iclass/statistics.py

```diff
@@ -18,13 +18,28 @@
 """
 
 import os
-from ctypes import *
+import sys
+from ctypes import c_int, c_char_p, c_float, byref
 
 import grass.script as gs
 
 try:
-    from grass.lib.imagery import *
-except ImportError as e:
+    from grass.lib.imagery import (
+        I_iclass_statistics_get_cat,
+        I_iclass_statistics_get_name,
+        I_iclass_statistics_get_color,
+        I_iclass_statistics_get_nbands,
+        I_iclass_statistics_get_ncells,
+        I_iclass_statistics_get_nstd,
+        I_iclass_statistics_get_min,
+        I_iclass_statistics_get_max,
+        I_iclass_statistics_get_range_min,
+        I_iclass_statistics_get_range_max,
+        I_iclass_statistics_get_mean,
+        I_iclass_statistics_get_stddev,
+        I_iclass_statistics_get_histo,
```

Comment:
Can you have these lines (only these ones, not the rest of the file) sorted with isort while we are adding them in?